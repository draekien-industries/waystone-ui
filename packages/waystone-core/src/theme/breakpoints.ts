export const breakpoints = ['480px', '768px', '1024px', '1200px'];

export default breakpoints;
