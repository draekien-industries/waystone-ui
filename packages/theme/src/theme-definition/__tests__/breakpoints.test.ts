import { breakpoints } from '../breakpoints';

describe('breakpoints', () => {
  it('should match snapshot', () => {
    expect(breakpoints).toMatchSnapshot();
  });
});
