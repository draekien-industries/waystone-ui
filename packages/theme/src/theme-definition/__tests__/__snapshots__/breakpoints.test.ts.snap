// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`breakpoints should match snapshot 1`] = `
[
  "480px",
  "768px",
  "1024px",
  "1200px",
]
`;
